components/page.scss

```diff
@@ -0,0 +1,4 @@
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/layout/page";
+@import "../scss/icons/icon";
```

Comment:
Icons are being imported with no variable set and no font is being imported, but a variable is being set.  Something seems amiss. 😄 Seems like fonts are needed, but not icons.